ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/WSQueryEndpoint.java

```diff
@@ -481,6 +504,20 @@ void start(
 
   }
 
+  interface IPullQueryPublisher {
```

Comment:
who implements this ? not following this change :(

Reply:
It's a functional interface. Previously there was one for both push and pull but now the signature changed since pull need the routing filters so I added an interface only for pull